### DISCUSSION TITLE
[정승용] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by jungdennis** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#65] [2024년도 과제 제출]
![image](https://github.com/user-attachments/assets/b353e357-f9e4-472f-9e8b-cfdddb0578b3)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다.

설문조사는 과제에 포함된 필수 항목입니다. 아래 링크에서 설문을 완료하시기 바랍니다.
https://forms.gle/DDwJCEf9Zbs3fxFi8

감사합니다.
